docs/DeveloperGuide.md

```diff
@@ -171,6 +171,50 @@ As such, the detailed descriptions for the Address Book subsystem above can be t
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### The onboarding guide
+
+This section describes the implementation of the onboarding guide subsystem.
+
+#### Accessing the Onboarding Guide
+The Onboarding Prompt, which leads to the onboarding guide only appears during first program run, which is determined by whether or not has there been any modification to the system. The onboarding guide can also be alternatively accessed through the menu bar.
+
+#### The `OnboardingWindow` Class
+To mimic the actual environment of Contax, the OnboardingWindow appears as a clone of the MainWindow, but with a different set of components aimed towards providing a comprehensive guide.
```

Comment:
```suggestion
To mimic the actual environment of ContaX, the OnboardingWindow appears as a clone of the MainWindow, but with a different set of components aimed towards providing a comprehensive guide.
```

minor nit write the name of the product as it is capitalised

Reply:
Can you also specify what the `components` refer? Are they UI components or logic components?

Reply:
> comprehensive guide

The onboarding guide is not supposed to be comprehensive, but a watered-down quick start guide.

Reply:
Fixed